src/Compilers/CSharp/Test/Syntax/Parsing/LambdaParameterParsingTests.cs

```diff
@@ -5687,7 +5687,38 @@ public void ScopedAsParameterName_05()
         }
 
         [Fact, WorkItem(63469, "https://github.com/dotnet/roslyn/issues/63469")]
-        public void ScopedAsParameterName_06()
+        public void ScopedAsParameterName_06_CSharp13()
+        {
+            string source = "(scoped scoped) => { }";
+            UsingExpression(source, TestOptions.Regular13);
```

Comment:
>UsingExpression(source, TestOptions.Regular13); [](http://example.com/codeflow?start=12&length=47)

If the parsing is expected to be unchanged, consider merging with the test below, and using `[Theory]`:
```
UsingExpression(source, useCSharp13 ? TestOptions.Regular13 : null);
```

Reply:
Done!